### PR TITLE
pmap to map in vcr yields correctly

### PR DESCRIPTION
**Pull request by kubamahnert**

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -73,8 +73,8 @@
       if self.class.metadata[:vcr]
         # replace parallel iterations with the serial one, since VCR can't handle parallel request matching correctly
         module Enumerable
-          def peach_with_index(*)
-            each_with_index
+          def peach_with_index(*, &y)
```
Comment (yenkins) [inline spec/spec_helper.rb]: Has the parameter name 'y' (UncommunicativeParameterName)

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -73,8 +73,8 @@
...
+            each_with_index &y
```
Comment (yenkins) [inline spec/spec_helper.rb]: Lint/AmbiguousOperator: Ambiguous block operator. Parenthesize the method arguments if it's surely a block operator, or add a whitespace to the right of the `&` if it should be a binary AND.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/110/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/929/ : FAILURE in 50s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1198/ : SUCCESS in 43s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/111/ : SUCCESS in 14s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/930/ : SUCCESS in 45s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1199/ : SUCCESS in 40s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/32/ : SUCCESS in 15s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/933/ : SUCCESS in 46s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1202/ : SUCCESS in 39s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1369/ : FAILURE in 56m 01s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/1132/ : FAILURE in 2m 18s

**Comment by Hahihula** [comment]
ok to merge

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/41/ : SUCCESS in 21s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/951/ : SUCCESS in 41s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1220/ : SUCCESS in 49s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1383/ : FAILURE in 1h 00m 54s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/1148/ : UNSTABLE in 36m 19s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/66/ : SUCCESS in 13s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1032/ : SUCCESS in 3m 51s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1301/ : SUCCESS in 2m 11s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1418/ : FAILURE in 31m 48s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-lcm-integration-tests-zuul-el7/3/ : SUCCESS in 36m 16s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/67/ : SUCCESS in 19s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1033/ : SUCCESS in 2m 48s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1302/ : SUCCESS in 3m 00s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1419/ : FAILURE in 28m 53s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-lcm-integration-tests-zuul-el7/4/ : SUCCESS in 33m 21s

**Comment by jimirocks** [comment]
The proposed change has been merged accidentally as part of another PR #1205

**Pull request closed without merging**